cmd/templates/main.go

```diff
@@ -1,3 +1,4 @@
+// nolint: gocritic
```

Comment:
Linteru se nelíbí kombinace `defer` a `exit` a nevím úplně jak to vyřešit. 🙂 Přidal jsem před exit ještě explicitně `tracer.Stop()`, je to tak ok?

![CleanShot 2022-03-02 at 11 47 24](https://user-images.githubusercontent.com/215660/156347929-22b3eb80-492e-4a99-97c3-80ab3c9a1f6d.jpg)

Reply:
Duplicitny kod mi nepride dobre riesenie, bude to niekto editovat a zabudne to niekde pridat.

Riesenim je spravit novu funkciu, `mainInt`, alebo `startHttpServer` .... je to jedno.

A v main bude len:
```
func main() {
  os.Exit(httpServer())
}

func httpServer() int {
  defer ...
}
```

Reply:
Myslim, ze este lepsie riesenie je z toho `main` vyhodit vsetko co nepotrebujeme:
https://github.com/keboola/keboola-as-code/commit/909d8c856eb9626ebc3e0c0a885d3621dbc61b43

Reply:
Upravil som tam rovno aj logger, aby sa aj chyby z datadogu vypisovali jednotne:
![image](https://user-images.githubusercontent.com/19371734/156389399-b33de996-8b4f-4af4-b3f4-860e59457e60.png)

Reply:
Suhlasis s tym? Schvalim tento PR, a poslem to ako svoj PR?